lib/utils/isUnique.test.js

```diff
@@ -0,0 +1,34 @@
+import { render } from '@folio/jest-config-stripes/testing-library/react';
+
+import { isUniqueField } from './isUnique';
+
+describe('isUniqueField', () => {
+  test('should return fieldValue marked if it is unique', () => {
```

Comment:
Can you use `it` instead of `test` for consistency purposes, please? in other components, we have used `it`

Reply:
fixed! Thanks for noticing it